CommonCompilerConfig.cmake

```diff
@@ -187,6 +187,7 @@ if(MSVC)
 		BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
 		BOOST_NO_WREGEX
 		_SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING
+		_SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING
```

Comment:
can you revert this and leave it to #846 ?

Reply:
Oh, forgot to remove that before submitting....cannot compile locally without it.